GameData/B9_Aerospace_HX/Parts/Structure_HX/model_hpd1.cfg

```diff
@@ -258,30 +135,79 @@ PART
                 loop = false
             }
         }
+		power_classic
+		{
+			AUDIO
+			{
+				name = light
+				channel = Ship
+				clip = sound_rocket_spurts
+				volume = 0.05 0.0
+				volume = 1.0 2.0
+				pitch = 0.0 0.2
+				pitch = 1.0 0.8
+				loop = true
+			}
+		}
+		power_plasma
+		{
+			AUDIO
+			{
+				name = medium
+				channel = Ship
+				clip = sound_rocket_hard
+				volume = 0.05 0.0
+				volume = 1.0 1.0
+				pitch = 0.0 1.0
+				pitch = 1.0 1.5
+				loop = true
+			}
+			AUDIO
+			{
+				name = jet
+				channel = Ship
+				clip = sound_jet_deep
+				volume = 0.0 0.0
+				volume = 0.2 0.5
+				volume = 0.5 1.0
+				pitch = 0.0 0.4
+				pitch = 0.5 0.6
+				loop = true
+			}
+		}
     }
 
     MODULE
     {
         name = MultiModeEngine
-        primaryEngineID = ClosedCycle
-        secondaryEngineID = HybridPlasma
+        primaryEngineID = SupraLox
+        secondaryEngineID = SupraNuclear
     }
-
+	
```

Comment:
???

Reply:
fancy new engine mode names to distinguish them and their characteristics from the typical ClosedCycle and Nuclear. 
* SupraLox: Beyond LFO convention
* SupraNuclear: Beyond NTR convention

Reply:
I was talking about the tab added to an empty line